src/main/java/frc/robot/components/TrigonXboxController.java

```diff
@@ -21,86 +21,68 @@
 
     public TrigonXboxController(int port) {
         super(port);
-        buttonA = new JoystickButton(this, Button.kA.value);
-        buttonB = new JoystickButton(this, Button.kB.value);
-        buttonX = new JoystickButton(this, Button.kX.value);
-        buttonY = new JoystickButton(this, Button.kY.value);
-        leftBumper = new JoystickButton(this, Button.kBumperLeft.value);
-        rightBumper = new JoystickButton(this, Button.kBumperRight.value);
-        leftStickButton = new JoystickButton(this, Button.kStickLeft.value);
-        rightStickButton = new JoystickButton(this, Button.kStickRight.value);
+        a = new JoystickButton(this, Button.kA.value);
+        b = new JoystickButton(this, Button.kB.value);
+        x = new JoystickButton(this, Button.kX.value);
+        y = new JoystickButton(this, Button.kY.value);
+        leftBumper = new JoystickButton(this, Button.kLeftBumper.value);
+        rightBumper = new JoystickButton(this, Button.kRightBumper.value);
+        leftStickButton = new JoystickButton(this, Button.kLeftStick.value);
+        rightStickButton = new JoystickButton(this, Button.kRightStick.value);
 
         backButton = new JoystickButton(this, Button.kBack.value);
         startButton = new JoystickButton(this, Button.kStart.value);
         rumbleAmount = -1;
         notifier = new Notifier(this::notifierPeriodic);
     }
 
-    public JoystickButton getButtonA() {
-        return buttonA;
+    public JoystickButton getABtn() {
+        return a;
     }
 
-    public JoystickButton getButtonB() {
-        return buttonB;
+    public JoystickButton getBBtn() {
+        return b;
     }
 
-    public JoystickButton getButtonX() {
-        return buttonX;
+    public JoystickButton getXBtn() {
+        return x;
     }
 
-    public JoystickButton getButtonY() {
-        return buttonY;
+    public JoystickButton getYBtn() {
+        return y;
     }
 
-    public JoystickButton getLeftBumper() {
+    public JoystickButton getLeftBumperBtn() {
```

Comment:
leftBumper not LeftBumperbutton

Reply:
not possible, as there's a function with that name in the parent class of type `boolean`